app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,51 @@
+class InvitesController < ApplicationController
+  before_action :set_invite, only: %i[destroy update]
+  before_action :set_user_to_id, only: :create
+  before_action :set_current_user_id, only: %i[index create]
+
+  attr_reader :invite, :user_to, :current_user_id
+
+  def index
+    @invites = Invite.where(user_from_id: current_user_id)
+  end
+
+  def create
+    @invite = Invite.new(user_from_id: current_user_id, user_to_id: user_to, account_id: params[:account_id])
+    if invite.valid?
+      invite.save && redirect_to(account_invites_url, notice: 'Invite have made.')
+    else
+      redirect_to account_invites_url, notice: 'Invite haven\'t been made'
+    end
+  end
+
+  def destroy
```

Comment:
I would put lower (after update method) - as a last method in public interface
(index, show, new, create, edit, update, delete) - I assume would be the correct order